### ISSUE TITLE
Evaluation Problem 

### ISSUE THREAD

**Issue by YangLi1221** [thunlp/HNRE#3]
I have downloaded your source code and trained without any change but I can not achieve the score from paper. The AUC score just 0.2966.
I wonder whether there is any tricks when I use your source code to train and to evaluate.

**Comment by Perfec-Yu** [comment]
I guess you are talking about the PCNN model. Indeed when training PCNN we used some sort of pretraining tricks as stated in our paper, and we have presented the pretrained encoder as vectors for initiation in this repo (see initial_pcnn file), so there would not be such wayward results. Would you give more details about experiments you carried out like parameter settings etc.

**Comment by YangLi1221** [comment]
Yeah, I use PCNN model to train. And the following is my parameter settings which is the same as your paper:
Batch size: 160
Learning rate: 0.2
Hidden size: 230
dropout : 0.9 ( its score is even worse...)
epoch: 40

**Comment by YangLi1221** [comment]
Did the dropout should change to 0.1, I see you noting it as keep prob and finally I get great score on it

**Comment by Perfec-Yu** [comment]
Sounds good.

**Comment by JunKuang-algo** [comment]
> Did the dropout should change to 0.1, I see you noting it as keep prob and finally I get great score on it

Hi, @UsamiLee 

I'd like to ask you what the auc score you got finally?
